system/settings/rbp.xml

```diff
@@ -27,6 +27,21 @@
           <control type="edit" format="integer" />
         </setting>
       </group>
+      <group id="3">
+        <setting id="videoplayer.supportmvc" type="boolean" label="38110" help="38111">
+          <level>2</level>
+          <default>true</default>
+          <control type="toggle" />
+        </setting>
+        <setting id="videoplayer.framepacking" type="boolean" label="38112" help="38113">
```

Comment:
enabling frame packed stereo modes is IMO a setting that would belong to "System -> Video" and not "Video -> acceleration". Also why does it depend on MVC support? Can't it be used for regular 3D modes as well to get a nicer looking GUI? I know that it, at least for the PI, only makes sense in combination, but if we think more general, it's IMO more a system setting (could also be used for 3D pictures if we ever add support for these).